### ISSUE TITLE
Voximplant.VideoView stacked [Android only]

### ISSUE THREAD

**Issue by Linoa65** [voximplant/react-native-voximplant#57]
Hello !

I actually implement the video inside my app and the communication works good. The only problem is for android, on iOS it works perfectly. When a video call is made, on my screen I have my remote view in full screen and my local view (camera) in the bottom left corner.

```jsx
<View style={[styles.littleCamera, styles.maxIndex]}>
    <Voximplant.VideoView
         style={[styles.imageBackground]}
         videoStreamId={this.state.videoLocalId}
         scaleType={Voximplant.RenderScaleType.SCALE_FILL}/>
 </View>

<View style={[styles.containerCamera, styles.minIndex]}>
    <Voximplant.VideoView
        style={[styles.imageBackground]}
        videoStreamId={this.state.videoRemoteId}
        scaleType={Voximplant.RenderScaleType.SCALE_FILL}/>
 </View>
```

maxIndex style is zIndex: 999 and minIndex style is zIndex: -999.

Everytime, my little camera is always behind my remote video stream. It seems on Android, the view is managed differently. Even I invert the order of my VideoViews, behavior is the same. My remote stream will always be on top of my local stream. On video loading, when screen is black, my local stream stay over my remote screen, but when it is fully loaded, remote stream goes on top of my local stream, every-times, only on Android.

If you need information, ask me :)

Best regards,
Florient

**Comment by YuliaGrigorieva** [comment]
Hello!

Thank you for reporting the issue!

Please provide the following information for further investigation:
1. react native version
2. Voximplant React Native SDK version
3. Information about the android device that issue is reproduced on (android version, device manufacture/model)

Best regards,
Yulia Grigorieva

**Comment by Linoa65** [comment]
Hello !

"react-native": "0.57.4"
"VoxImplant SDK": "1.4.0"

I tested it on emulator with a Google Pixel 2 XL (api 28), and on Huawei P10 Lite (api 26)

Thanks :)
Florient

**Comment by Linoa65** [comment]
Hello !

Some news about this issue ?

**Comment by YuliaGrigorieva** [comment]
Hello!

Thank you for the provided information, I was able to reproduce the issue. 
It seems that the issue is Android 7+ specific and we are investigating it.

I will let you know as soon as the fix is available.

Best regards,
Yulia Grigorieva

**Comment by YuliaGrigorieva** [comment]
Hello, @Linoa65 !

We have just released Voximplant React Native SDK 1.5.0 that includes the fix for this issue: new `showOnTop` prop for VideoView component. 
 
Please upgrade the SDK in your project and use this commit in demo project as a reference: https://github.com/voximplant/react-native-demo/commit/8bc47f81f0baca93f3f35202c8439ecc984c6d99

Please let me know if the issue is resolved on your side.

Best regards,
Yulia Grigorieva

**Comment by YuliaGrigorieva** [comment]
Closing the issue for now, feel free to reopen if the issue is reproduced in your project.
